tests/playwright/src/specs/image-manifest-smoke.spec.ts

```diff
@@ -0,0 +1,144 @@
+  test.describe
+    .serial('Image Manifest Validation - Simple Containerfile', () => {
+      test('Build the image using cross-arch build (simple )', async () => {
+        await playExpect(imagesPage.heading).toBeVisible();
+
+        const buildImagePage = await imagesPage.openBuildImage();
+        await playExpect(buildImagePage.heading).toBeVisible();
+        const dockerfilePath = path.resolve(__dirname, '..', '..', 'resources', 'test-containerfile');
+        const contextDirectory = path.resolve(__dirname, '..', '..', 'resources');
+
+        imagesPage = await buildImagePage.buildImage(imageNameSimple, dockerfilePath, contextDirectory, architectures);
+        await playExpect.poll(async () => await imagesPage.waitForImageExists(manifestNameSimple)).toBeTruthy();
+        await playExpect.poll(async () => await imagesPage.countRowsFromTable()).toBe(4);
+        await imagesPage.toggleImageManifest(manifestNameSimple);
+        await playExpect.poll(async () => await imagesPage.countRowsFromTable()).toBe(2);
+      });
+      test('Check Manifest details', async () => {
+        const imageDetailsPage = await imagesPage.openImageDetails(manifestNameSimple);
+
+        await Promise.all(
+          architectures.map(async architecture => {
+            await playExpect(imageDetailsPage.tabContent).toContainText(architecture);
+          }),
+        );
+        await playExpect(imageDetailsPage.backLink).toBeVisible();
+        await imageDetailsPage.backLink.click();
+      });
+      test('Delete Manifest', async ({ page }) => {
+        await deleteImageManifest(page, manifestNameSimple);
+      });
+    });
+  test.describe
+    .serial('Image Manifest Validation - Complex Containerfile', () => {
+      test('Build the image using cross-arch build (complex)', async ({ page }) => {
+        await playExpect(imagesPage.heading).toBeVisible();
+
+        const buildImagePage = await imagesPage.openBuildImage();
+        await playExpect(buildImagePage.heading).toBeVisible();
+        const dockerfilePath = path.resolve(
+          __dirname,
+          '..',
+          '..',
+          'resources',
+          'alphine-hello',
+          'alphine-hello-containerfile',
+        );
+        const contextDirectory = path.resolve(__dirname, '..', '..', 'resources', 'alphine-hello');
+
+        try {
+          imagesPage = await buildImagePage.buildImage(
+            imageNameComplex,
+            dockerfilePath,
+            contextDirectory,
+            architectures,
+          );
+        } catch {
+          skipTests = true;
+          await deleteImageManifest(page, manifestNameComplex);
+          if (isWindows && isCI && hypervisor === 'wsl') {
+            console.log('Building cross-architecture images with the WSL hypervisor is not working yet');
+            test.fail();
+          } else {
+            throw new Error('Failed to build image manifest');
```

Comment:
I think you should propagate the error from try block here. so it can get "bubbled up". And it should be assertion error, not standard error.